data/human/deep jobs.txt

```diff
@@ -820,156 +788,145 @@ phrase "stranded consultant dropoff payment"
 		attributes "deep"
 		attributes "urban"
-		not attributes "station" "moon"
+		not attributes "station"
 	on accept
 		dialog phrase "stranded consultant pickup"
 	on visit
 		dialog phrase "generic passenger on visit"
 	on complete
-		payment 200000 35000
-		"reputation: Deep Security" += 0.5
+		payment 10000 300
+		"reputation: Deep Security" += 1
 		dialog phrase "stranded consultant dropoff payment"
-	on fail
-		"reputation: Deep Security" -= 4
-	on abort
-		"reputation: Deep Security" -= 10
 
 mission "Transport Consultants (Deep Security) [1]"
-	name "Transport Deep Security consultants to <planet>"
+	name "Transport Deep Security consultants"
 	job
 	repeat
-	description "Transport consultants whose schedules have been ruined by Deep Security to <destination>. Payment is <payment>."
+	description "Transport <fare> whose schedules have been ruined by Deep Security to <destination>. Payment is <payment>."
 	passengers 1 8
 	to offer
 		random < 20
 		has "deep security consultants"
-		"reputation: Deep Security" >= 100
+		"reputation: Deep Security" >= 60
 	source
 		attributes "deep"
-		not attributes "station" "moon"
+		not attributes "station"
 	destination
+		distance 2 7
 		attributes "deep"
 		attributes "urban"
-		not attributes "station" "moon"
+		not attributes "station"
 	on accept
 		dialog phrase "stranded consultant pickup"
 	on visit
 		dialog phrase "generic passenger on visit"
 	on complete
-		payment 250000 40000
-		"reputation: Deep Security" += 0.5
+		payment 10000 300
+		"reputation: Deep Security" += 1
 		dialog phrase "stranded consultant dropoff payment"
-	on fail
-		"reputation: Deep Security" -= 4
-	on abort
-		"reputation: Deep Security" -= 10
 
 
 mission "Urgent Deep Sec Package"
-	name "Rush Deep Security Delivery to <planet>"
+	name "Rush Deep Security Delivery"
 	repeat
 	deadline 1 1
-	description "Deliver <cargo> to <destination> by <date>. Payment is <payment>."
+	description "Deliver a Deep Security package to <destination> by <date>. Payment is <payment>."
 	minor
 	cargo "deep security package" 1
 	to offer
-		random < 2
+		random < 3
+		has "deep security reveal"
+		"reputation: Deep Security" >= 40
 	source
 		attributes "deep"
-		not attributes "station" "moon"
 	destination
+		distance 6 7
 		attributes "deep"
 		not planet "Valhalla"
-		not attributes "station" "moon"
+		not attributes "station"
 	on offer
-		dialog `As you are walking through the spaceport, a man approaches you and flashes a Deep Security ID. "Greetings Captain <last>. We need someone to deliver this to <destination> as soon as possible."`
+		dialog `As you are walking through the spaceport, a man approaches you and flashes a Deep Security ID. "Greetings Captain <last>. We need someone to deliver this to <destination> by <date>, taking the most direct route possible."`
 	on visit
 		dialog phrase "generic cargo on visit"
 	on complete
-		payment 200000 30000
-		"reputation: Deep Security" += 0.5
+		payment 170000 10000
```

Comment:
I think these last three jobs still have way too high base payments. 170k or 450k guaranteed for 6-7 jumps of not refueling doesn't match the difficulty, as these are really only difficult to first time you take them, then after that you should be set to complete any of them without issue. For comparison, 150k is what you get for tracking down and killing a medium warship bounty, and 450k is what you get for taking on a Marauder V bounty, which is a Marauder medium warship and two lights.

```suggestion
		payment 30000 1000
```